### PR TITLE
Add Dirt Plant

### PR DESCRIPTION
**Pull request by Skizzles**
Additions:
Adds Dirt Plant
Magical Essence to use 8 to create 2 dirt

Changes:
Some above imports that are redundant. (If not wanted can be removed)

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java [hunk 1]
```diff
@@ -55,6 +19,23 @@
 import me.mrCookieSlime.Slimefun.cscorelib2.skull.SkullItem;
 import me.mrCookieSlime.Slimefun.cscorelib2.updater.GitHubBuildsUpdater;
 import me.mrCookieSlime.Slimefun.cscorelib2.updater.Updater;
+import org.bstats.bukkit.Metrics;
+import org.bukkit.*;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java:23]: **NEVER** use star imports.

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java [hunk 2]
```diff
@@ -1,42 +1,6 @@
...
-import org.bukkit.potion.PotionEffectType;
-
-import io.github.thebusybiscuit.exoticgarden.items.Crook;
-import io.github.thebusybiscuit.exoticgarden.items.CustomFood;
-import io.github.thebusybiscuit.exoticgarden.items.ExoticGardenFruit;
-import io.github.thebusybiscuit.exoticgarden.items.FoodRegistry;
-import io.github.thebusybiscuit.exoticgarden.items.GrassSeeds;
-import io.github.thebusybiscuit.exoticgarden.items.Kitchen;
-import io.github.thebusybiscuit.exoticgarden.items.MagicalEssence;
+import io.github.thebusybiscuit.exoticgarden.items.*;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java:3]: **NEVER** use star imports.

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java [hunk 3]
```diff
@@ -55,6 +19,23 @@
...
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java:37]: **NEVER** use star imports.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java (hunk 1 above)
Comment (Skizzles) [reply src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java]: Added back

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java (hunk 2 above)
Comment (Skizzles) [reply src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java]: Added back as well

File: src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java (hunk 3 above)
Comment (Skizzles) [reply src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java]: Added this back as well

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**